File: backend/src/main/java/de/rubeen/bsc/entities/web/LoginUser.java
package de.rubeen.bsc.entities.web;

import org.springframework.data.annotation.Id;

public class LoginUser {
    private String email, password;

    LoginUser(String email, String password) {
        this.email = email;
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }
}

File: backend/src/main/kotlin/de/rubeen/bsc/entities/web/LoginUser.kt
package de.rubeen.bsc.entities.web

class LoginUser {
    var id: Int? = null
    var email: String? = null
    var password: String? = null

    constructor()

    constructor(email: String, password: String) {
        this.email = email
        this.password = password
    }
}
